sdk/typescript/src/client/client.ts

```diff
@@ -551,14 +553,46 @@ export class IotaClient {
     /**
      * Return the latest IOTA system state object on networks supporting protocol version `>= 5`.
      * These are networks with node software release version `>= 0.11`.
+     *
+     * You probably want to use `getSupportedIotaSystemState` instead.
      */
     async getLatestIotaSystemStateV2(): Promise<IotaSystemStateSummary> {
-        return await this.transport.request({
+        return await this.transport.request<IotaSystemStateSummary>({
             method: 'iotax_getLatestIotaSystemStateV2',
             params: [],
         });
     }
 
+    /**
+     * Return the latest supported IOTA system state object.
```

Comment:
I would explaina bit more what this does and why is useful, i would personally be much more verbose here